content/Simulation_Methods_Mikaela_Pisani.md

```diff
@@ -0,0 +1,73 @@
+
+# Simulation Methods
+Simulation can be used for emulating situations. It is used to simulate systems to be studied. This can be applied in any kind of systems, banks, supermarkets, biological systems, traffic, etc. And also is used to generate data with certain characteristics, either to complete missing information, to generate more data “similar” to the original one or to generate “fake” data to be detected by your model. 
```

Comment:
```suggestion
Simulation can be used for emulating situations. It is used to simulate systems to be studied. This can be applied in any kind of system, banks, supermarkets, biological systems, traffic, etc. And also is used to generate data with certain characteristics, either to complete missing information, to generate data more “similar” to the original one, or to generate “fake” data to be detected by your model. 
```